src/openrct2/ride/coaster/meta/LIMLaunchedRollerCoaster.h

```diff
@@ -24,5 +24,6 @@ constexpr const RideTypeDescriptor LIMLaunchedRollerCoasterRTD =
                      RIDE_TYPE_FLAG_PEEP_CHECK_GFORCES | RIDE_TYPE_FLAG_ALLOW_MULTIPLE_CIRCUITS),
     SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_TRAIN, RIDE_COMPONENT_TYPE_TRACK, RIDE_COMPONENT_TYPE_STATION }),
     SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_RESTRAINTS_STUCK_CLOSED) | (1 << BREAKDOWN_RESTRAINTS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION) | (1 << BREAKDOWN_BRAKES_FAILURE)),
+    SET_FIELD(LiftData, { SoundId::Null, 4, 6 }),
```

Comment:
This is the only one that is null, but not `5, 5`. Should it?

(Note for future: sounds weird for this coaster to have no sound)